frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.ts

```diff
@@ -16,10 +16,11 @@ import { WindowRef } from '../../../_services/native-window.service';
 
```

Comment:
Let's remove this entire file from the PR so that all the changes stay within the scope of this issue. After the prettier is in place we'll go back over individual files and apply it to them.

Reply:
@briandavidson - I removed these changes and reverted.